netdisco/discoverables/webos_tv.py

```diff
@@ -13,7 +13,21 @@ def info_from_entry(self, entry):
 
     def get_entries(self):
         """Get all the LG WebOS TV device uPnP entries."""
-        return self.find_by_device_description({
-            "deviceType": "urn:dial-multiscreen-org:device:dial:1",
-            "friendlyName": "[LG] webOS TV"
```

Comment:
I'm not sure I really like this hardcoding since this forces user to name the device exactly like that, when usually the device name itself looks like "[LG] webOS TV UH7507" so by default unless users are careful they might miss the [docs](https://home-assistant.io/components/media_player.webostv/) that specify "all LG WebOS Smart TV’s within your network will be auto-discovered if your TV network name is set to [LG] webOS TV".

With the new approach that I've added I don't care about device name as I only look for manufacturer and model.